python/contactflowhelper/lambda_function.py

```diff
@@ -0,0 +1,114 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+import json
+import random
+
+def lambda_handler(event, context):
+    response = {}
+    
+    if event['Details']['Parameters']['function'] == 'freeform_math':
+        response.update({'answer':str(freeform_math(event))})
+        
+    elif event['Details']['Parameters']['function'] == 'random_number':
+        response.update({'answer':str(random_number(event))})
+        
+    elif event['Details']['Parameters']['function'] == 'increment_1':
+        response.update({'answer':str(increment_1(event))})
+        
+    elif event['Details']['Parameters']['function'] == 'increment_n':
+        response.update({'answer':str(increment_n(event))})
+        
+    elif event['Details']['Parameters']['function'] == 'random_choice':
+        response.update({'answer':str(random_choice(event))})
+        
+    elif event['Details']['Parameters']['function'] == 'replace_text':
+        response.update({'answer':str(replace_text(event))})
+        
+    elif event['Details']['Parameters']['function'] == 'split_text':
+        response.update(split_text(event))
+        
+    elif event['Details']['Parameters']['function'] == 'strip_text':
+        response.update({'answer': str(strip_text(event))})
+        
+    elif event['Details']['Parameters']['function'] == 'upper_text':
+        response.update({'answer': str(upper_text(event))})
+        
+    elif event['Details']['Parameters']['function'] == 'lower_text':
+        response.update({'answer': str(lower_text(event))})
+        
+    else:
+        response.update({'result':'fail'})
+        response.update({'reason':'no valid function passed'})
+    
+    response.update({'result':'success'})
+    return response
+    
+def freeform_math(event):
+    result = eval(event['Details']['Parameters']['expression'])
+    return result
+    
+def random_number(event):
+    result = round(random.uniform(int(event['Details']['Parameters']['start']),int(event['Details']['Parameters']['end'])))
+    return result
+    
+def increment_1(event):
+    result = int(event['Details']['Parameters']['base'])+1
+    return result
+    
+def increment_n(event):
+    result = int(event['Details']['Parameters']['base'])+int(event['Details']['Parameters']['increment'])
+    return result
+    
+def random_choice(event):
+    choice_list = event['Details']['Parameters']['list'].split(',')
+    result = random.choice(choice_list)
+    return result
+    
+def replace_text(event):
+    do_replace = event['Details']['Parameters']['text_string']
+    result = do_replace.replace(event['Details']['Parameters']['replace_this'], event['Details']['Parameters']['with_this'])
+    return result
+    
+def split_text(event):
```

Comment:
This only splits once? Why not make this dynamic and split however many times the character is present?

Reply:
Added the option to specify max splits.